client/components/resubscribeThrasher.tsx

```diff
@@ -80,14 +79,13 @@ const getThrasher =
 								},
 							}}
 						>
-							Support The Guardian with a recurring contribution
-							of your choice. You can use your existing payment
-							details so setting it up only takes a minute.
+							From just $1, or a little more on a regular basis,
```

Comment:
ello 👋  ... is this always supposed to be a dollar value?

Reply:
We don't have easy access to the payment currency in the thrasher unfortunately. It does make an API call to fetch existing payment options, but this doesn't include the currency.

Iona is fine with this and has advised that we already do this in a number of other places where regionalisation is impossible or not worth the dev effort. (Her style guide also includes this as guidance.)